### PR TITLE
Fix-It Days: Placeholders 5 (Brent)

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Removes placeholders from the Authorization section, part 2.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-729831](https://oktainc.atlassian.net/browse/OKTA-729831)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/client-secret-rotation-key/main/index.md](https://github.com/okta/okta-developer-docs/blob/86508b9d6537aa46470fa79899cb13ac7836007b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fclient-secret-rotation-key%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d53042c8-26ea-4e4e-a277-f077ad50a278) | 79  | 74  | 72  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md](https://github.com/okta/okta-developer-docs/blob/86508b9d6537aa46470fa79899cb13ac7836007b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-ciba%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3b033229-279b-4cb5-beee-bf68c2012f6d) | 87  | 71  | 65  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md](https://github.com/okta/okta-developer-docs/blob/86508b9d6537aa46470fa79899cb13ac7836007b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-native-sso%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2ffc82ed-df25-4394-96df-466346b60475) | 32  | 72  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/86508b9d6537aa46470fa79899cb13ac7836007b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/efee16c8-a6ba-4731-b422-1a6c5538ad05) | 94  | 80  | 71  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md](https://github.com/okta/okta-developer-docs/blob/86508b9d6537aa46470fa79899cb13ac7836007b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Frequest-user-consent%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3c0b0f63-1995-4d4a-81a3-5af9a7473479) | 81  | 70  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md](https://github.com/okta/okta-developer-docs/blob/86508b9d6537aa46470fa79899cb13ac7836007b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fset-up-token-exchange%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/baa598a9-df4f-407b-9f8a-430afeaf7e73) | 58  | 75  | 69  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.489ae1de-466c-4b85-b840-6d708d0de2c7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/index.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b2579080-e2e6-43cf-a9ab-4cc1e11ca264) | 100  | 88  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/java/createclientsecretjwt.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjava%2Fcreateclientsecretjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4e2a3998-d2d1-4702-9ed2-4c25c5fa8b1c) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/java/createprivatekeyjwt.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjava%2Fcreateprivatekeyjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2724684a-cf0f-4ef9-aa15-7465e5999804) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/js/createclientsecretjwt.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjs%2Fcreateclientsecretjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/cbc249cb-5e8a-4e1e-b8f6-f3ee9b0f50fa) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/js/createprivatekeyjwt.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjs%2Fcreateprivatekeyjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e05e3fe6-df5e-4849-9232-5c7d694da1e1) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/client-secret-rotation-key/main/index.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fclient-secret-rotation-key%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a8a8e4ef-f339-4111-9fcf-756d48466f25) | 79  | 74  | 72  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-ciba%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4fe09058-679a-4c10-9e3a-701e7880604d) | 87  | 71  | 65  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-native-sso%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a2033231-0f93-4118-8519-c97c8690983c) | 32  | 72  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8787467a-34c6-4c29-9b7e-dc28603ed112) | 94  | 80  | 71  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Frequest-user-consent%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dedcb0b3-398a-41a3-bd7b-3cdf3e615c3f) | 81  | 70  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md](https://github.com/okta/okta-developer-docs/blob/2c8d2e0aae079325790c33eecd18072783ce7eab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fset-up-token-exchange%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/348c4591-1996-4656-95b6-e473c5d64dda) | 58  | 75  | 69  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.bc8d4430-db40-4e45-a7e2-78c354ef6c54?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/23369a0b-c744-4380-b395-56ec4b3249e8) | 100  | 88  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/java/createclientsecretjwt.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjava%2Fcreateclientsecretjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d18426bd-f410-4a53-a311-10081eeec728) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/java/createprivatekeyjwt.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjava%2Fcreateprivatekeyjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/741c13bb-c987-4b2e-ae1b-b8cf079b854e) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/js/createclientsecretjwt.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjs%2Fcreateclientsecretjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a3cc1a01-47ba-45b9-95de-2adbf597c31f) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/js/createprivatekeyjwt.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjs%2Fcreateprivatekeyjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bafc8c6b-014d-4f2f-9f55-fe7b8a2f46e7) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/client-secret-rotation-key/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fclient-secret-rotation-key%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2eba66e3-ab91-4b79-aaa3-1e28dce4d5b3) | 79  | 74  | 72  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-ciba%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7c3b7569-ebc3-4914-be8c-9288cff5a5f9) | 87  | 71  | 65  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-native-sso%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7767a80e-a5e9-451d-abdd-eeb1028ef8e4) | 32  | 72  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7336df11-c0b1-401b-b906-ed02ab193a10) | 94  | 80  | 71  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-tokens-dynamic%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0e96f05a-f2e8-410a-98e4-f5484f123d0e) | 84  | 66  | 63  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-tokens-groups-claim/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-tokens-groups-claim%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/61bfc1a0-b3a0-4667-8b09-bf54f285ed20) | 80  | 56  | 70  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-tokens-returned-from-okta%2Fmain%2Findex.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7551d39d-f119-476e-94be-4e555daf976d) | 92  | 75  | 67  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-tokens-static%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d5d371e4-6d3c-4b9b-adbc-a21a378cad98) | 86  | 66  | 69  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Frequest-user-consent%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/508bc38b-210e-46e9-bd61-6cba21c8a225) | 81  | 70  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md](https://github.com/okta/okta-developer-docs/blob/796d89f9c2e63e35c2f09619602183c13a6d6110/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fset-up-token-exchange%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/91010220-fdb4-4001-b992-73467a6b18c5) | 58  | 75  | 69  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.31e0ce86-3369-4afc-b1df-2f935573c36e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/index.md [hunk 1]
```diff
@@ -32,15 +32,15 @@ Which JWT type that you use depends on the client authentication method configur
 
 > **Note:** To create a client application and specify either the `client_secret_jwt` or `private_key_jwt` authentication method, see the [Add OAuth 2.0 Client Application](/docs/reference/api/apps/#add-oauth-2-0-client-application) API reference section. To change the client authentication method of an existing app, see the [Update the Client Authentication Method](/docs/reference/api/apps/#update-the-client-authentication-method) API reference section.
 
-> **Tip:** Don't know which method is configured for your client app? You can quickly check the settings by doing a GET to `https://${yourOktaDomain}/api/v1/apps/${appId}`.
+> **Tip:** Don't know which method is configured for your client app? You can quickly check the settings by doing a GET to `https://{yourOktaDomain}/api/v1/apps/{appId}`.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/index.md:35]:
```suggestion
> **Tip:** If you don't know which method is configured for your client app, you can quickly check the settings by doing a GET to `https://{yourOktaDomain}/api/v1/apps/{appId}`.
```

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6264fc9a-8298-4eb7-b9c2-abaae87724a4) | 100  | 88  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/java/createclientsecretjwt.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjava%2Fcreateclientsecretjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/85916866-58d3-430c-9445-775a02cbd288) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/java/createprivatekeyjwt.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjava%2Fcreateprivatekeyjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ca5e189e-b741-45f7-a19c-a6164b4b112f) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/js/createclientsecretjwt.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjs%2Fcreateclientsecretjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/167f3554-39e9-4a84-80ca-e5c8609abf70) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-self-signed-jwt/main/js/createprivatekeyjwt.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-self-signed-jwt%2Fmain%2Fjs%2Fcreateprivatekeyjwt.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/437bfa6e-18b9-4a90-855e-5bbeacdd6754) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/client-secret-rotation-key/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fclient-secret-rotation-key%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0759fc49-7751-4494-8451-84e7a5a44a2b) | 79  | 74  | 72  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-ciba%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/51618a67-eb2a-409a-80f2-17acb4206800) | 87  | 71  | 65  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-native-sso%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f2cb4614-d859-46da-9c3c-e85f7b508c31) | 32  | 72  | 83  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-authz-server/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-authz-server%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/43d86071-4b31-48c6-a142-3c49fd3a10ec) | 94  | 80  | 71  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-tokens-dynamic/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-tokens-dynamic%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3c884d09-f37d-4a34-9034-ce90127ba9ae) | 84  | 66  | 63  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-tokens-groups-claim/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-tokens-groups-claim%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b078b255-1bb7-447d-813c-53951a066b64) | 80  | 56  | 70  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-tokens-returned-from-okta%2Fmain%2Findex.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/db9124a6-00c9-4731-a243-9d65c52708fb) | 92  | 75  | 67  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/customize-tokens-static/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustomize-tokens-static%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3018d779-7b2d-4737-be4b-f26c06bc04c0) | 86  | 66  | 69  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Frequest-user-consent%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3de1e068-592a-4948-9377-185c90bd658b) | 81  | 70  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md](https://github.com/okta/okta-developer-docs/blob/86a0a057227f5af0b7f7c078cc7fcfd7ffb2f3b4/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fset-up-token-exchange%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8624ad99-a6b2-4181-bc82-0f19d5715de4) | 58  | 75  | 69  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e5974f18-2fee-45d0-bd42-1cf8117ad567?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**